config.py

```diff
@@ -49,7 +50,6 @@
     'Dx Aliquot Post Enrichment (clean) v1.0', 'Dx Aliquot Post Enrichment (clean) v1.1',
     'Dx Post Enrichment PCR & clean up v1.0', 'Dx Post Enrichment PCR & clean up v1.1',
     'Dx Aliquot Post Enrichment PCR (clean) v1.0', 'Dx Aliquot Post Enrichment PCR (clean) v1.1',
-    'Dx Library Prep & Target Enrichment Magnis v1.0'
```

Comment:
Was the removal of 'Dx Library Prep & Target Enrichment Magnis v1.0' intended? Because I can't see it is necessary for [Make email server configurable.].

Reply:
Yes, I synced the config with the dev server. Commit msg should have reflected that as well. Just like the change on line 25.